packages/xo-web/src/common/intl/messages.js

```diff
@@ -1426,6 +1426,7 @@ const messages = {
   vmLimitsLabel: 'VM limits',
   resourceSet: 'Resource set',
   resourceSetNone: 'None',
+  selectCreator: 'Select creator',
```

Comment:
What do you think of `Select user`?
Since in reality you choose a user who will become the creator of the VM.